src/qrcode_scripts/qrcoderasp.py

```diff
@@ -0,0 +1,42 @@
+import cv2
+from signin import *
+#select source image as video capture
+cap = cv2.VideoCapture(0)
+#sign in to post/get API calls
+signinpy()
+#detect QE in current video capture
+detector = cv2.QRCodeDetector()
+VideoOn =True
+while VideoOn:
+    
+    #detect and box the QR code detection
+    _, img = cap.read()
```

Comment:
Change this to a more descriptive name, also what is `_`? Why is it needed?
```suggestion
    _, QRImage = cap.read()
```

Reply:
The underscore is needed because cap.read() returns other information that should be ignored. You can refer to the next site to have a better understanding of the underscore in python: https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc